src/EmotiBitWiFiHost.h

```diff
@@ -75,6 +85,9 @@ class EmotiBitWiFiHost
 
 	~EmotiBitWiFiHost();
 	int8_t begin();
+	void EmotiBitWiFiHost::getAvailableSubnets();
```

Comment:
Is there a reason it's not just `void getAvailableSubnets()`? why are we scoping it in the class it belongs to?